server/apps/users/views.py

```diff
@@ -131,3 +133,73 @@ def get(request):
         serializer = TodoSerializer(todos, many=True)
 
         return Response(serializer.data)
+
+
+class CreateTodoView(APIView):
```

Comment:
Use `Generic views`

Reply:
https://www.django-rest-framework.org/api-guide/generic-views/#generic-views

Reply:
Implemented in latest commit